doc/developer-notes.md

```diff
@@ -124,6 +124,23 @@ can be very difficult to track down. Compiling with -DDEBUG_LOCKORDER (configure
 CXXFLAGS="-DDEBUG_LOCKORDER -g") inserts run-time checks to keep track of which locks
 are held, and adds warnings to the debug.log file if inconsistencies are detected.
 
+**Memory Profiling**
+
+Bitcoin Unlimited can be compiled with the libtcmalloc allocation library and
```

Comment:
Adding a '... on Linux ...' in the documentation makes it clearer to people running on other platforms that this diagnostic feature isn't currently supported on their platforms.
That's why I raised this point.